src/utils/dns.js

```diff
@@ -74,11 +74,12 @@ const processAnswer = (type, answer) => {
     return answer;
 };
 
-const performLookupJson = async (domain, type, endpoint) => {
+const performLookupJson = async (domain, type, endpoint, cdflag) => {
     // Build the query URL
     const query = new URL(endpoint.endpoint);
     query.searchParams.set('name', domain);
     query.searchParams.set('type', type.toLowerCase());
+    query.searchParams.set('cd', cdflag);
```

Comment:
```suggestion
    query.searchParams.set('cd', cdflag.toString().toLowerCase());
```
Just enforcing consistency in the value here